run_release.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+npm_version_params=$*
+
+current_branch=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
I would love to see some sort of a yes/no question before doing the rest of the code to prevent accidental changes. Not a blocker but it would be neat to make a git tag for that version too maybe.

Reply:
+1. The `npm version` below automatically creates a new tag.